src/figuresClasses.ts

```diff
@@ -1,11 +1,78 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea: () => number;
+}
+
+export class Triangle implements Figure {
+  shape: string = 'triangle';
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
```

Comment:
The properties `a`, `b`, and `c` are declared in the constructor but are not assigned to `this.a`, `this.b`, and `this.c`. This will result in an error when trying to access these properties in the `getArea` method. You should assign them to `this.a`, `this.b`, and `this.c` within the constructor.